### DISCUSSION TITLE
AdhocFilters: Support mixed data source

### DISCUSSION THREAD

**Discussion by Grubana** [grafana/grafana#39396] [Dashboards, Panels & Visualizations]
**What happened**:
![19-09-_2021_22-52-38](https://user-images.githubusercontent.com/1355287/133942703-54e9912a-8853-42cf-aa55-9f2a8e272ca6.gif)
When adding an adhoc filter within a mixed datasource panel, the filter is not getting applied to the queries.

**What you expected to happen**:
Filters are applied to all adhoc enabled datasources within the panel

**How to reproduce it (as minimally and precisely as possible)**:
Create a table panel with a mixed datasource.
Try to filter for a value within the table.

**Anything else we need to know?**:
Looks like the adhoc filter is tied to the mixed datasource type, instead of the concrete datasources used in the panel config

**Environment**:
- Grafana version: v8.1.3 (a61f38238c)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
